config/config.go

```diff
@@ -76,6 +79,8 @@ func application(cfg *Config) (*kingpin.Application, *string) {
 
 	app.Flag("id", "ID of this pmm-agent. [PMM_AGENT_ID]").
 		Envar("PMM_AGENT_ID").PlaceHolder("</agent_id/...>").StringVar(&cfg.ID)
+	app.Flag("port", "Agent local API port. [PMM_AGENT_PORT]").
```

Comment:
Please rename it to `listen-port` or something else. We have another port next line.